src/form_components/repeating_group.coffee

```diff
@@ -121,6 +121,11 @@ FormRenderer.Views.ResponseFieldRepeatingGroup = Backbone.View.extend
       @views.push view
 
     @$el.html JST['partials/repeating_group'](@)
+
+    @$el.removeClass('is_truncated')
+    if (@model.entries.length && @model.entries[0].formComponents.length > 1)
```

Comment:
If a question is skipped, `entries.length` will be `[]`. If it's not skipped, the first entry will have `formComponents` which is an array of the fields within. If there are at least 2 fields, we truncate the question. This isn't changing the height currently (which is static and hides overflow), but just applies the class so we draw the ripped paper effect at the bottom of our existing truncation height.

Reply:
Agree - it looks good enough at 90% to merge this PR